README.md

```diff
@@ -1,12 +1,12 @@
 # The Graphics Gremlin - a Retro ISA Video Card
 
-The Graphics Gremlin is an FPGA-based ISA video card specifically designed to emulate certain old video standards. This initial release emulates the original IBM PC monochrome graphics adapter (MDA) as well as the original IBM color graphics adapter (CGA). Since the logic is defined by the bitstream loaded into the FPGA, new emulations may be available in the future to support other video standards.
+The Graphics Gremlin is an FPGA-based ISA video card specifically designed to emulate certain old video standards. This initial release emulates the original IBM PC hercules graphics card (HGC) as well as the original IBM color graphics adapter (CGA). Since the logic is defined by the bitstream loaded into the FPGA, new emulations may be available in the future to support other video standards.
```

Comment:
it still supports MDA, so let's mention that too.

Reply:
I will, however, HGC really is an evolution of MDA, it is implicitly compatible with it.